pkg/api/v1/schema_swagger_generated.go

```diff
@@ -218,35 +235,38 @@ func (Features) SwaggerDoc() map[string]string {
 
 func (FeatureState) SwaggerDoc() map[string]string {
 	return map[string]string{
-		"":        "Represents if a feature is enabled or disabled",
+		"":        "Represents if a feature is enabled or disabled\n+k8s:openapi-gen=true",
 		"enabled": "Enabled determines if the feature should be enabled or disabled on the guest\nDefaults to true\n+optional",
 	}
 }
 
 func (FeatureAPIC) SwaggerDoc() map[string]string {
 	return map[string]string{
+		"":               "+k8s:openapi-gen=true",
 		"enabled":        "Enabled determines if the feature should be enabled or disabled on the guest\nDefaults to true\n+optional",
 		"endOfInterrupt": "EndOfInterrupt enables the end of interrupt notification in the guest\nDefaults to false\n+optional",
 	}
 }
 
 func (FeatureSpinlocks) SwaggerDoc() map[string]string {
 	return map[string]string{
+		"":          "+k8s:openapi-gen=true",
 		"enabled":   "Enabled determines if the feature should be enabled or disabled on the guest\nDefaults to true\n+optional",
 		"spinlocks": "Retries indicates the number of retries\nMust be a value greater or equal 4096\nDefaults to 4096\n+optional",
 	}
 }
 
 func (FeatureVendorID) SwaggerDoc() map[string]string {
 	return map[string]string{
+		"":         "+k8s:openapi-gen=true",
```

Comment:
Not something you introduced, but we are also missing comments on the struct (I think in the spec there are comments, they would just need to be copied over).

Reply:
yup, i'll fix